warrior/Framework/Utils/encryption_utils.py

```diff
@@ -54,8 +53,23 @@ def get_key(encoded_key):
             MYFILE = Tools.__path__[0]+os.sep+"admin"+os.sep+'secret.key'
             with open(MYFILE, 'r') as myfileHandle:
                 encoded_key = myfileHandle.read()
+            if not encoded_key:
```

Comment:
Hi @ryadaval-warrior,

It will be better if the code that generates a random key is taken out into a separate function which is called whenever needed rather than repeating code .

Thanks,
Sanika